lib/jekyll/drops/document_drop.rb

```diff
@@ -9,8 +9,16 @@ class DocumentDrop < Drop
 
       def_delegator :@obj, :next_doc, :next
       def_delegator :@obj, :previous_doc, :previous
-      def_delegator :@obj, :relative_path, :path
       def_delegators :@obj, :id, :output, :content, :to_s, :relative_path, :url
+      def_delegators :@obj, :name, :name
+
+      def path
+        if @obj.is_a? Page
+          @obj.path
+        else
+          @obj.relative_path
+        end
+      end
```

Comment:
* Please use () in all `if` statements.  Never skip the () in an if statement as it can be ambiguous to a human.